jupyter-js-widgets/package.json

```diff
@@ -54,8 +54,7 @@
     "webpack": "^1.12.11"
   },
   "dependencies": {
-    "backbone": "1.2.0",
-    "bootstrap": "^3.3.5",
+    "backbone": "^1.2.3",
```

Comment:
No, 1.1.0 (see https://github.com/ipython/ipython-components/blob/master/bower.json#L5)  Immediately post big split, pre 4.0, we upgraded which caused major headaches because of backwards incompat. changes in backbone.